Common Word Definitions.md

```diff
@@ -0,0 +1,33 @@
+Common Word Definitions
```

Comment:
This file isn't linked from anywhere, and seems to be a little bit deficient. Are you planning on putting a link in sometime in the future, and just working on it on the side for now?